web/src/SidebarItem.tsx

```diff
@@ -42,6 +43,8 @@ class SidebarItem {
 
     this.name = res.name ?? ""
     this.isTiltfile = !!res.isTiltfile
+    this.isTest =
+      (res.localResourceInfo && !!res.localResourceInfo.isTest) || false
```

Comment:
slightly simpler as `!!res.localResourceInfo?.isTest`

Reply:
nice. ugh i swear i will NEVER understand js.

Reply:
oh, this is a new programming language nerd thing! typescript only added it in 2020

https://en.wikipedia.org/wiki/Safe_navigation_operator

i think it started out in Groovy/C# and has taken over a ton of programming languages over the last decade